### DISCUSSION TITLE
Indent Html/jsx Tag

### DISCUSSION THREAD

**Discussion by bwafi** [LazyVim/LazyVim#1125] [Q&A]
when i enter tag html in jsx file

example:

`<div>|</div>`

when i press enter:

```
<div>
|</div>
```

i want :

`<div>|</div>`

to :

```
<div>
  |
</div>
```

**Comment by skoch13** [comment]
[This](https://www.reddit.com/r/neovim/comments/10kah18/comment/j5rueyw/?utm_source=share&utm_medium=web2x&context=3) might be the solution you're looking for:

```
vim.keymap.set('n', '<CR>', '@="m`o<C-V><Esc>``"<CR>')
vim.keymap.set('n', '<S-CR>', '@="m`O<C-V><Esc>``"<CR>')
```

**Comment by itzcodex24** [comment]
Not sure if this issue is still an active one, but please find the solution here: https://github.com/LazyVim/LazyVim/discussions/2020#discussioncomment-7581193
